src/net/sourceforge/kolmafia/FamiliarData.java

```diff
@@ -575,8 +575,8 @@ private static void parseSoup(final String responseText) {
       if (fam == null) continue;
 
       var data = json.getJSONObject(key);
-      fam.setSoupWeight(data.getInt("times"));
-      var attrs = data.getJSONArray("attr").toList().stream().map(String.class::cast).toList();
+      fam.setSoupWeight(data.getInteger("times"));
```

Comment:
```suggestion
      fam.setSoupWeight(data.getIntValue("times"));
```
avoid boxing / unboxing